README.md

```diff
@@ -1,167 +1,140 @@
 # OSDe2e
- 
+
 [![GoDoc](https://godoc.org/github.com/openshift/osde2e?status.svg)](https://godoc.org/github.com/openshift/osde2e)
- 
-Comprehensive testing solution for Service Delivery
- 
-## Purpose
-Provide a standard for testing every aspect of the Openshift Dedicated product. Use data derived from tests to inform release and product decisions.
- 
-## Setup
- 
-Log into OCM, then go here to obtain an [OpenShift Offline Token].
- 
-A properly setup Go workspace using **Go 1.18+ is required**.
- 
-Install dependencies:
-```
-# Install dependencies
-$ go mod tidy
-```
- 
-Set OCM_TOKEN environment variable:
-```
-$ export OCM_TOKEN=<token from step 1>
-```
- 
-## The `osde2e` command
- 
-The `osde2e` command is the root command that executes all functionality within the osde2e repo through a number of subcommands.
- 
-### Running from source
- 
-To run osde2e locally, first build the binary (do this after all changes) by running `make build`. The resulting binaries will be in `./out/`.
- 
-Once built, you can invoke osde2e by running `./out/osde2e`.
- 
-A common workflow is having a local script that combines these steps and the config. Example:
- 
-```bash
-#!/usr/bin/env bash
-make build
- 
-GINKGO_SKIP="" \
-CLEAN_CHECK_RUNS="3" \
-POLLING_TIMEOUT="5" \
-OCM_TOKEN="[OCM token here]" \
-./out/osde2e test --configs "prod,e2e-suite"
-```
- 
-Another example:
-```bash
-#!/usr/bin/env bash
+
+## Introduction
+
+A comprehensive test framework used for Service Delivery to test all aspects of
+Managed OpenShift Clusters ([OpenShift Dedicated]). The data generated by
+the different test coverage is used to inform product releases and decisions.
+
+OSDe2e key features are:
+
+* Portable test framework that can run anywhere to validate end to end test workflows
+  * Run locally from a developers workstation or from a CI application
+* Supports create/delete different cluster deployment types
+  * ROSA, ROSA Hosted Control Plane (e.g. HyperShift), OSD
```

Comment:
Should this mention support for GCP and Azure as well?

Reply:
:+1: thanks for pointing that out, will add it.